### PR TITLE
Loadbalancer listener module

### PR DESCRIPTION
**Pull request by anton-sidelnikov**
slightly reworked loadbalancer.py
added integration test
added listener module
added listener info module

### CODE REVIEW

**Comment by anton-sidelnikov** [comment]
@gtema somehow in zuul always failing TASK [loadbalancer : Create Router for ELB], can you look at that? In 447 integration test passed

**Review by gtema — CHANGES_REQUESTED** [review]
would you please generate random name for the integration test to avoid conflicts (what was reason for previous failures). Example is in other scenarios

File: tests/integration/targets/loadbalancer/tasks/main.yaml [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+    loadbalancer:
+      cloud: "{{ test_cloud }}"
+  block:
+     - name: Set random prefix
+      set_fact:
+        prefix: "{{ 99999999 | random | to_uuid | hash('md5') }}"
+
+    - name: Set initial facts
+      set_fact:
+        network_name: "{{ ( prefix + '_my_test_network_lb') }}"
```
Comment (gtema) [inline tests/integration/targets/loadbalancer/tasks/main.yaml:12]: please as a suffix also use useful name to understand where orphans are coming from (i.e. acc-lb - AnsibleCollectionCloud-LoadBalancer)

File: tests/integration/targets/loadbalancer/tasks/main.yaml [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+    - name: Get loadbalancer by Name
+      loadbalancer_info:
+        name: "{{ lb.loadbalancer.name }}"
+      register: res
+
+    - name: assert result
+      assert:
+        that:
+          - res is success
+          - res.otc_loadbalancers is defined
```
Comment (gtema) [inline tests/integration/targets/loadbalancer/tasks/main.yaml:130]: please in the module rename otc_loadbalancers output to simply loadbalancers

**Review by gtema — APPROVED** [review]

**Comment by anton-sidelnikov** [comment]
regate

**Comment by anton-sidelnikov** [comment]
regate

**Comment by gtema** [comment]
regate

**Pull request merged**